src/Interactive/CsiCore/project.json

```diff
@@ -1,6 +1,9 @@
 ﻿{
   "dependencies": { },
   "frameworks": {
+    "netstandard1.6": {
+      "imports": "portable-net452"
+    },
```

Comment:
Chatting with NuGet team, since they use Newtonsoft.Json comments should work here. I'll update this quick.